### PR TITLE
[lighthouse] add reference for "title" audit

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?
- updated the reference template
- created reference for "doesn't have title element" audit

**Fixes:** N/A

**Target Live Date:** 2018-03-09

- [x] This has been reviewed and approved by @kaycebasques
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/tools/lighthouse/audits/title.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+# Document Doesn't Have A Title Element  {: .page-title }
+
+## Overview {: #overview }
+
+Note: This audit occurs in both the Accessibility and SEO categories of your
+Lighthouse report.
+
+### Accessibility {: #accessibility }
+
+The document title gives screen readers users an overview of a page's purpose.
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/title.md]: `screen readers users` => `users of screen readers`?

File: src/content/en/tools/lighthouse/audits/title.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+[HIR]: https://support.google.com/webmasters/answer/80407
+
+Tips for creating great titles:
+
+* Make them descriptive and concise. Avoid vague descriptions like `Home`.
+* Avoid keyword stuffing. It's not helpful to users, and search engines may
+  mark the page as spam.
+* Avoid repeated or boilerplate titles.
+* It's OK to brand your titles, but do it concisely.
+* Be careful about disallowing search engines. Your page can still be indexed,
```
Comment (rviscomi) [inline src/content/en/tools/lighthouse/audits/title.md]: This one doesn't seem like it belongs with the others, on the subject of document titles. WDYT?

File: src/content/en/tools/lighthouse/audits/title.md (hunk 1 above)
Comment (rviscomi) [reply src/content/en/tools/lighthouse/audits/title.md]: I guess it gives _everyone_ an overview of the page's purpose, but maybe say that it's especially useful to screen reader users who have limited context about the page.

File: src/content/en/tools/lighthouse/audits/title.md (hunk 2 above)
Comment (kaycebasques) [reply src/content/en/tools/lighthouse/audits/title.md]: Yes, good point. I was just taking queues from the webmaster article... but on second thought it doesn't really make sense here.

**Pull request merged**